### DISCUSSION TITLE
After ssh is connected to the server, the entered commands are not displayed on the interface

### DISCUSSION THREAD

**Discussion by MervynYan** [Eugeny/tabby#8971] [Q&A] [answered]
When I used tabby's ssh function to connect to the server, the keyboard input command was not displayed on the tabby page, but was displayed after entering the press enter

**Comment by Clem-Fern** [comment, marked as answer]
Hey @MervynYan,

It could be related to this issue #8884.
As workaround, you can try to switch to `xterm (canvas)` in `Settings > Terminal > Rendering > Frontend`.

**Reply by MervynYan** [reply]
Thanks.
